rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -338,6 +338,14 @@ public int getReceiptsCacheSize() {
         return configFromFiles.getInt("cache.receipts.max-elements");
     }
 
+    public int getBtcBlockStoreCacheSize() {
+        return configFromFiles.getInt("cache.btcBlockStore.size");
+    }
+
+    public int getBtcBlockStoreCacheDepth() {
+        return configFromFiles.getInt("cache.btcBlockStore.depth");
```

Comment:
There are two things that you can check here (or in the caller)

1. That `depth <= size`

2. That `depth >= 4000` or some other constant. You can add a warning log in case that it fails, there's no reason to fail. But a lower value sounds problematic

Reply:
Ok, I will add some warning.